client-contract/src/Types.sol

```diff
@@ -13,6 +13,7 @@ using CBORDecoder for bytes;
 using BigIntCBOR for CommonTypes.BigInt;
 using BigIntCBOR for bytes;
 using FilecoinCBOR for CBOR.CBORBuffer;
+using FilecoinCBOR for bytes;
```

Comment:
Remove this entire file and use the functionality provided in the lib here: https://github.com/Zondax/filecoin-solidity/blob/master/contracts/v0.8/cbor/MarketCbor.sol